projects/core/src/occ/adapters/user/occ-user-cost-centers.adapter.ts

```diff
@@ -25,8 +26,15 @@ export class OccUserCostCenterAdapter implements UserCostCenterAdapter {
   ) {}
 
   loadActiveList(userId: string): Observable<EntitiesModel<CostCenter>> {
+    // @see feature-libs/asm/root/interceptors/user-id.interceptor.ts
```

Comment:
These comments are nice, but they bring a maintenance overhead.
Perhaps looking up the token references is enough for devs to understand the usage?